code/database/migrations/2022_08_18_033727_add_default_roles.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Spatie\Permission\Models\Role;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
```

Comment:
Desde PHP 7 se agrego manera de tipar los retornos y parámetros, antes usábamos PHPdoc para documentar esto  actualmente ya no es necesario y en su lugar debemos optar por documentar esto con la sitaxi que el leguaje brinda mas nos ayudara a indicarle a PHP que tipos se espera, esto también ayuda a los IDE a brindarnos una mejor ayuda. 

El comentario de descripción también se depreca para impulsar los nombres descriptivos en los nombres de nuestros métodos, en este caso no podemos definir un nombre ya que esto lo definen las migraciones, pero cualquier dev de Laravel sabe para que es el método up en una migración. 

```suggestion
    public function up(): void
```

lo mismo para el método down.

Reply:
https://www.php.net/manual/es/language.types.declarations.php